Core/Newsletter2Go/Bootstrap.php

```diff
@@ -8,7 +8,7 @@
  */
 class Shopware_Plugins_Core_Newsletter2Go_Bootstrap extends Shopware_Components_Plugin_Bootstrap
 {
-    const VERSION = '4.1.19';
+    const VERSION = '4.1.20';
```

Comment:
Our latest_Version of the Connector is already 4.1.20 and is also released. We need to use the Version 4.1.21 for this plugin.